wxt_cidrbot/git_webhook_handler.py

```diff
@@ -352,6 +352,9 @@ def send_approved_message(self, installation_id, json_string):
 
         self.logging.debug("review message: %s", review_message)
 
+        token = self.dynamo.get_repo_keys(self.room_id, repo_name)
```

Comment:
Change self.room_id to room_id like it's used on line 358

There's no self. variable present here